src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/EventCounterListener.cs

```diff
@@ -195,87 +209,29 @@ private void ExtractAndPostMetric(string eventSourceName, IDictionary<string, ob
                             EventCountersInstrumentationEventSource.Log.EventCounterRefreshIntervalLessThanConfigured(actualInterval, this.options.RefreshIntervalSecs);
                         }
                     }
-                    else if (key.Equals("Count", StringComparison.OrdinalIgnoreCase))
```

Comment:
Why did you remove this?

Reply:
It was not used. So why reading it from the payload.

Reply:
Oh I get it now -thanks.